site/app/templates/Vue.twig

```diff
@@ -1,6 +1,6 @@
 <script> 
     $('document').ready(() => {
-        window.submitty.render('[data-vuemount]', "{{ page }}",  {{ args | json_encode(constant("JSON_FORCE_OBJECT")) | raw() }});
```

Comment:
Should this change be made in a separate bugfix PR?

Reply:
I don't get why this change is showing up in this PR, it was merged into main with https://github.com/Submitty/Submitty/pull/10780

Reply:
Oh I just needed to merge in main through github (it didnt work when I did it locally?)